src/services/API/sqs.js

```diff
@@ -0,0 +1,50 @@
+import { SQSClient, SendMessageCommand } from "@aws-sdk/client-sqs";
+
+const REGION = "ap-south-1";
+const queueURL = "https://sqs.ap-south-1.amazonaws.com/111766607077/EventQueue";
+const sqsClient = new SQSClient({
+  region: REGION,
+  credentials: {
+    accessKeyId: process.env.VUE_APP_AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.VUE_APP_AWS_SECRET_KEY,
+  },
+});
+
+async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, doesUserExist, authType) {
+    const messageBody = [
+      {
+        dateTime: Date.now().toString(),
+        purpose: {
+          type: purpose,
+          subType: purposeParams,
+          params: {
+            platform: redirectTo,
+            id: redirectID,
+          },
+        },
+        authType: authType,
+        user: {
+          values: userID,
+          userDataValidated: doesUserExist,
+        },
+      },
+    ];
+    const params = {
+      MessageBody: JSON.stringify(messageBody),
+      QueueUrl: queueURL,
+    };
+
+    try {
+      const data = await sqsClient.send(new SendMessageCommand(params));
+      console.log("Success, message sent. MessageID:", data.MessageId);
```

Comment:
Hmm.. console log.

Reply:
Shouldn't I do this? Should I use Logger here?

Reply:
We don't have a JS logger no?

Reply:
there isn't a default logger but a package.

Reply:
No point of doing console.log here as it will try to log to the browser, which is useless for us when someone else is using the app. I think in the future we can include sentry tracking so we can track all errors remotely but not required right now.

Also @sudheshna-donthineni there's a package that we use in plio-frontend which automatically removes all the `console.log` statements when the code is built for production. The name of the package is `babel-plugin-transform-remove-console`.
Take a look at the plio-frontend to see how it's being configured.

Reply:
Added to my to-do.

Reply:
Added to my to-do.